docs/1.2. Behaviour.md

```diff
@@ -4,30 +4,31 @@ The Sink Metadata Processing API provides a mechanism to change the settings ass
 
 ## Senders
 
-`/senders/{senderId}/media-profiles` allows the client to pass an array of [Media Profiles](1.0.%20Overview.md#media-profile) to a Sender and let the underlying component control which it choose exactly which format to use. A client can also pass a single Media Profile to specify the exact format.
+The `/media-profiles` endpoint allows the client to pass an array of [Media Profiles](1.0.%20Overview.md#media-profile) to a Sender and let the underlying component to choose exactly which format from the set it will use. A client can pass a single Media Profile to specify the exact format.
```

Comment:
```suggestion
The `/media-profiles` endpoint allows the client to pass an array of [Media Profiles](1.0.%20Overview.md#media-profile) to a Sender and let the underlying implementation to choose exactly which format from the set it will use. A client can pass a single Media Profile to specify the exact format.
```